packages/datatrak-web/src/utils/pwa.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+
+function getPWADisplayMode() {
```

Comment:
Bit less ambiguous to treat PWA as a “word” in camel case. (Unfortunate that web APIs capitalise entire abbreviations 🥲)

```suggestion
function getPwaDisplayMode() {
```